### ISSUE TITLE
Can someone make a simple exe for CPU users?

### ISSUE THREAD

**Issue by bil80** [xinntao/Real-ESRGAN#781]
Hi Developer Team, i'm enjoying the exe GPU version for windows, but i would appreciate a CPU-optimized executable for users without GPU access. If anyone has it! Could you please share it with me? Thanks in advance!

**Comment by SA-j00u** [comment]
> Hi Developer Team, i'm enjoying the exe GPU version for windows, but i would appreciate a CPU-optimized executable for users without GPU access. If anyone has it! Could you please share it with me? Thanks in advance!

not optimized but you can run ncnn resizer from releases on CPU with swiftshader
https://github.com/xinntao/Real-ESRGAN/issues/792#issuecomment-2093994912
just put all dll-s to folder with esrgan-ncnn-vulkan.exe 
(also pth2ncnn.exe works with it too
or you can try replace onnx2ncnn.exe and ncnnoptimize.exe with no shared versions https://github.com/Tencent/ncnn/releases that works without vulkan)

pytorch version can be runed on CPU too
but it's not simple exe at all.. (just 4GB of pyth00n dependency ballast)

**Comment by bil80** [comment]
Yes, this is exactly what I needed! It works perfectly. I'm using the prebuilt version from the link you shared: https://github.com/google/gfbuild-swiftshader/releases/download/github%2Fgoogle%2Fgfbuild-swiftshader%2F8f075627d16bdd2a8d861e9d81df541f5cf68e2e/gfbuild-swiftshader-8f075627d16bdd2a8d861e9d81df541f5cf68e2e-Windows_x64_Debug.zip

Thank you man :)

**Comment by NS-Clone** [comment]
> Yes, this is exactly what I needed! It works perfectly. I'm using the prebuilt version from the link you shared: https://github.com/google/gfbuild-swiftshader/releases/download/github%2Fgoogle%2Fgfbuild-swiftshader%2F8f075627d16bdd2a8d861e9d81df541f5cf68e2e/gfbuild-swiftshader-8f075627d16bdd2a8d861e9d81df541f5cf68e2e-Windows_x64_Debug.zip
> 
> Thank you man :)

debug version may by several times slower!
